README.markdown

```diff
@@ -223,10 +235,13 @@ For UIKit view controllers, consider grouping lifecycle, custom accessors, and I
 
 ### Unused Code
 
-Unused (dead) code, including Xcode template code and placeholder comments should be removed.
+Xcode template code and placeholder comments should **not** be removed.
+
+Other unused (dead) code should be removed and should never reach `development` branch. You can keep commented code in your branch or even in a feature branch, but `development` branch should be clean from commented code. If a future feature needs some code that has been removed, it can always be retrieved using git. Mark your commit name accordingly when removing code if it is susceptible to be used in the future, it will make it easier to cherry pick.
```

Comment:
I hope we are all in agreement with this. I have come across a ton of dead code (in the editor, in merchant, in achievements...) and I think we need to start cleaning up our code. If we are implementing some new feature and come across some dead code in the same file, we should remove it.